### PR TITLE
Fixture/mm/header/button

### PR DESCRIPTION
**Pull request by mmarkiewicz1**

### CODE REVIEW

File: src/core/StyledButton/StyledButton.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        extend: 'contained'
+    },
+    text: {
+        fontFamily: [IBMPlexSansLight]
+    },
+    secondary: {
+        extend: 'outlined',
+        borderColor: '#ffffff',
+        '&:hover': {
+            borderColor: '#ffffff',
```
Comment (solleOlle) [inline src/core/StyledButton/StyledButton.jsx:46]: why? we have it on 44 line

File: src/stories/core/TopMenuButton/TopMenuButton.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    title: 'Example/TopMenuButton',
+    component: TopMenuButton
+};
+
+const Template = args => <TopMenuButton {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+    //label: 'TopMenuButton',
+    //color: 'primary'
```
Comment (solleOlle) [inline src/stories/core/TopMenuButton/TopMenuButton.stories.jsx:14]: please remove it

**Review by solleOlle — APPROVED** [review]

File: src/core/ButtonWrapper/ButtonWrapper.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { Button, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+    root: {
+        fontSize: props => props.fontSize,
+        fontWeight: props => props.fontWeight,
+        color: props => props.color
+    }
```
Comment (sergeevds) [inline src/core/ButtonWrapper/ButtonWrapper.jsx:12]:
```
root: ({ fontSize, fontWeight, color }) => ({
        fontSize,
        fontWeigh,
        color
 })
```

File: src/core/TopMenuButton/TopMenuButton.jsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+TopMenuButton.propTypes = {
+    children: PropTypes.node,
+    variant: PropTypes.oneOf(['contained', 'outlined']),
+    color: PropTypes.oneOf(['primary', 'secondary']),
+    disableElevation: PropTypes.bool,
+    disableRipple: PropTypes.bool,
+    size: PropTypes.number,
+    text: PropTypes.string
+};
```
Comment (sergeevds) [inline src/core/TopMenuButton/TopMenuButton.jsx:52]: you use only `size` and `text` props.

File: src/core/StyledButton/StyledButton.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            {...props}
+        >
+            {text || children}
+        </ButtonWrapper>
+    );
+};
+
+StyledButton.propTypes = {
+    children: PropTypes.node,
+    text: PropTypes.string
```
Comment (sergeevds) [inline src/core/StyledButton/StyledButton.jsx:73]: StyledButton has to support all props from ButtonWrapper

File: src/core/StyledButton/StyledButton.jsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    contained: {
+        extend: 'root',
+        backgroundColor: '#ea3d26',
+        '&:hover': {
+            backgroundColor: '#c43525'
+        }
+    },
+    outlined: {
+        extend: 'root',
+        borderColor: '#ea3d26',
```
Comment (sergeevds) [inline src/core/StyledButton/StyledButton.jsx:30]: can we add the color in theme?

**Review by sergeevds — CHANGES_REQUESTED** [review]

**Review by wmosienski — CHANGES_REQUESTED** [review]

File: src/core/TopMenuButton/TopMenuButton.jsx [hunk 7]
```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
+import React from 'react';
+import { makeStyles, Typography } from '@material-ui/core';
+import { ReactComponent as IconMail } from 'assets/images/icon_mail-0222.svg';
+import StyledButton from '../StyledButton';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(({ size }) => ({
```
Comment (wmosienski) [inline src/core/TopMenuButton/TopMenuButton.jsx:8]: here you have theme, not props

File: src/core/TopMenuButton/TopMenuButton.jsx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    icon: {
+        width: props => props.size * 4,
+        height: props => props.size * 4,
+        '@media (min-width: 900px)': {
```
Comment (wmosienski) [inline src/core/TopMenuButton/TopMenuButton.jsx:12]: since you added breakpoints to the theme, you could use it here

File: src/theme/theme.jsx [hunk 9]
```diff
@@ -1,9 +1,9 @@
 import { createTheme } from '@material-ui/core/styles';
-import { IBMPlexSansLight } from './typography';
+import { fonts } from './typography';
 
 const theme = createTheme({
     typography: {
-        fontFamily: [IBMPlexSansLight.fontFamily, 'Arial', 'sans-serif'].join(','),
+        // fontFamily: [IBMPlexSansLight.fontFamily, 'Arial', 'sans-serif'].join(','),
```
Comment (wmosienski) [inline src/theme/theme.jsx:6]: you know what to do :)

**Review by wmosienski — CHANGES_REQUESTED** [review]

File: src/core/TopMenuButton/TopMenuButton.jsx (hunk 8 above)
Comment (mmarkiewicz1) [reply src/core/TopMenuButton/TopMenuButton.jsx:12]: done :+1:

File: src/theme/theme.jsx (hunk 9 above)
Comment (mmarkiewicz1) [reply src/theme/theme.jsx]: done

File: src/core/TopMenuButton/TopMenuButton.jsx (hunk 7 above)
Comment (mmarkiewicz1) [reply src/core/TopMenuButton/TopMenuButton.jsx]: done

File: src/theme/theme.jsx [hunk 10]
```diff
@@ -19,23 +17,39 @@ const theme = createTheme({
...
+            },
+            black: {
+                main: '#000000'
+            }
         }
     },
     breakpoints: {
         values: {
-            xs: 0,
+            xs: 400,
```
Comment (mmarkiewicz1) [inline src/theme/theme.jsx:40]: .

**Pull request merged**